views/lib/default.hbs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{userLanguage}}" timezone="{{currentTimezone}}">
+<html lang="{{#if userLanguage}} {{userLanguage}} {{else}}de{{/if}}" timezone="{{currentTimezone}}">
```

Comment:
I think this is not necessary here since language has default as 'de'

Reply:
I've noticed userLanguage is undefined.

Reply:
okay ;)